ironfish-cli/src/commands/accounts/notes.ts

```diff
@@ -37,9 +37,15 @@ export class NotesCommand extends IronfishCommand {
         [note],
         {
           amount: {
-            header: 'Amount ($IRON)',
+            header: 'Amount',
```

Comment:
It doesn't the denomination now, but I guess that's ok. $IRON here is denomination rather than the currency but this is fine.